README.md

```diff
@@ -125,6 +125,17 @@ Here are some pointers for anyone looking for mini-features and work items that
 
 We try not to be too OCD about coding style wars, but we do [have our own convention](https://github.com/reactiveui/ReactiveUI/blob/master/CONTRIBUTING.md) and [best design practices documented](https://github.com/reactiveui/rxui-design-guidelines) - please respect them and your pull-request experience will be much smoother. If you are using Visual Studio, please [install the rebracer plugin](https://visualstudiogallery.msdn.microsoft.com/410e9b9f-65f3-4495-b68e-15567e543c58) which will automatically apply the correct source formatting settings.
 
+We're also looking for people to assist with code reviews of ReactiveUI contributions. If you're experienced with any of the below technologies, you can join the team and receive notifications:
+
+ - [Android](https://github.com/orgs/reactiveui/teams/reviewers-android) reviewers
```

Comment:
Yeah, slip-up. I *think* it'll look better with `reviewers` in the link text. Let's try that.